manual-workflows/start.sh

```diff
@@ -11,11 +11,13 @@ $0 - Start a new Cromwell VM instance
 usage: $0 INSTANCE_NAME [--argument value]*
 
 arguments:
--h, --help         print this block and immediately exits
---project          GCP project name
---server-account   Email identifier of service account used by main Cromwell instance
---cromwell-conf    Local path to configuration file for Cromwell server. DEFAULT $SRC_DIR/cromwell.conf
---machine-type     GCP machine type for the instance. DEFAULT e2-standard-2
+-h, --help           print this block and immediately exits
```

Comment:
Suggest minor rewording to:
"prints this block and immediately exits" or "print this block and immediately exit"